run-unit-tests.sh

```diff
@@ -1,4 +1,9 @@
 npm run build:test
+# TODO: how to capture all tests with a single glob expression?
+for f in ./dist/node/**/**/*.unit-test.js; do
+  echo "Running $f"
+  node $f || exit 1;
+done
 for f in ./dist/node/**/*.unit-test.js; do
```

Comment:
These two for loops look the same to me, is there a mistake here?

Reply:
there's a double `**/**` in the first, and one `**` in the second. I originally wanted all nested subdirectories, but bash instead just treated `**` like `*` and just gave me one level of hierarchy. For the mina-signer tests I needed to include another level.

I just looked it up and learned how to make it behave the way I want (https://unix.stackexchange.com/a/197382), so I'll try that